### PR TITLE
Default to system ca-certificates.crt if none is specified

### LINKED ISSUE #10853: Email confirmation can't be send due to ssl certificate failure
### Expected behaviour

Sendmail should have send out the email.

### Actual behaviour

log from sidekiq
````
OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=error: certificate verify failed (unable to get local issuer certificate)
--
````

### Steps to reproduce the problem

Setup email delivery with gmail and ssl_verify peer and try to register a new account.

### Specifications

2.8.4

GSuite business

And I know I could turn off ssl_verify but I don't want to.

related to #10831

Edit:
Suggestion: Default https://github.com/tootsuite/mastodon/blob/22de24b8ca707d8eac26eab6615377a290ff1f4e/config/environments/production.rb#L89 to ``/etc/ssl/certs/ca-certificates.crt``.

### PR DESCRIPTION
**Pull request by SuperSandro2000**
I know you can set ``SMTP_CA_FILE``  but i think this is a better default if you enable ssl_verify peer.

Closes #10853

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Bump

File: config/environments/production.rb [hunk 1]
```diff
@@ -86,7 +86,7 @@
     :password             => ENV['SMTP_PASSWORD'].presence,
     :domain               => ENV['SMTP_DOMAIN'] || ENV['LOCAL_DOMAIN'],
     :authentication       => ENV['SMTP_AUTH_METHOD'] == 'none' ? nil : ENV['SMTP_AUTH_METHOD'] || :plain,
-    :ca_file              => ENV['SMTP_CA_FILE'].presence,
+    :ca_file              => ENV['SMTP_CA_FILE'].present? ? ENV['SMTP_CA_FILE'].presence : '/etc/ssl/certs/ca-certificates.crt',
```
Comment (ykzts) [inline config/environments/production.rb]:
```suggestion
    :ca_file              => ENV['SMTP_CA_FILE'].presence || '/etc/ssl/certs/ca-certificates.crt',
```

**Review by ykzts — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
Bump. This breaks the installer when you want to send email with smtp over tls.

**Comment by SuperSandro2000** [comment]
bump

**Comment by SuperSandro2000** [comment]
bump

**Comment by SuperSandro2000** [comment]
Bump

**Review by ClearlyClaire — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
@Gargron ping

**Comment by SuperSandro2000** [comment]
bump

**Pull request merged**